presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/StandardColumnMappings.java

```diff
@@ -279,7 +285,7 @@ public static ColumnMapping timeColumnMappingUsingSqlTime(ConnectorSession sessi
                     TIME,
                     (resultSet, columnIndex) -> {
                         Time time = resultSet.getTime(columnIndex);
-                        return toPrestoLegacyTimestamp(toLocalTime(time).atDate(LocalDate.ofEpochDay(0)), sessionZone);
+                        return toPrestoLegacyTimestamp(createTimestampType(3), toLocalTime(time).atDate(LocalDate.ofEpochDay(0)), sessionZone);
```

Comment:
If you want to rely on the fact `createTimestampType(3)` returns a constant, just use the (deprecated)
constant directly. If you do not want to rely on that -- capture `createTimestampType(3)` as a constant outside of the lambda.